### PR TITLE
Added solution for 98. Valid Binary Search Tree

### LINKED ISSUE #20: IsValidBST
please assign me this under hacktoberfest

### PR DESCRIPTION
**Pull request by srexrg**
Created Added Readme.md with isValid.java...Please merge the request with hacktober label

closes #20

### CODE REVIEW

**Comment by Tusharknwl** [comment]
@srexrg Please follow the [Directory Structure](https://github.com/godkingjay/LeetCode#directory-structure) and ensure that your files are named accordingly.

**Comment by srexrg** [comment]
Sorry..will make changes to it

**Comment by srexrg** [comment]
Made changes according to naming conventions,directory structure and also added cpp solution

**Review by godkingjay — CHANGES_REQUESTED** [review]
Have you tried running this code in LeetCode? I'm quite suspicious of this code as I only get errors, please add the code you entered and tested in LeetCode.

File: Medium/98.Valid BST/solution.java
Comment (godkingjay) [inline Medium/98.Valid BST/solution.java:1]: Have you tried running this code in LeetCode? I'm quite suspicious of this code as I only get errors, please add the code you entered and tested in LeetCode. It should have this structure:

```java
class Solution {
	/* ... */
    public boolean isValidBST(TreeNode root) {
    	/* ... */    
    }
	/* ... */
}
```

File: Medium/98.Valid BST/solution.cpp
Comment (godkingjay) [inline Medium/98.Valid BST/solution.cpp:1]: Have you tried running this code in LeetCode? I'm quite suspicious of this code as I only get errors, please add the code you entered and tested in LeetCode. It should have this structure:

```cpp
class Solution {
/* ... */
public:
    bool isValidBST(TreeNode* root) {
        /* ... */
    }
	/* ... */
};
```

**Comment by srexrg** [comment]
Ok will do it

**Comment by srexrg** [comment]
added leetcode sol cpp and java

**Review by godkingjay — APPROVED** [review]
Thank you for your contribution.

- [x] C++ Solution
- [x] Java Solution

---
**Closes:**
- #20

**Pull request merged**